anjali29/bio.html

```diff
@@ -0,0 +1,204 @@
+
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"><html><head><META http-equiv="Content-Type" content="text/html; charset=utf-8"></head>
+<body>
+ <div style="border-radius:5px">
+      <div style="position:fixed;height:40px;width:98%;background-color:#668284;margin-bottom:10px">
+         <p style="float:left;margin-left:20px;padding-bottom:10px;font-size:16px;font-family:Verdana,sans-serif;color:#ffffff">
+            ANJALI ASHOK ASADE
+         </p>
+         <a href="mailto:anjaliasade@gmail.com" target="_blank">
+            <p style="float:right;margin-right:20px;padding-bottom:10px;font-size:16px;font-family:Verdana,sans-serif;color:#ffffff">anjaliasade@gmail.com</p>
+         </a>
+      </div>
+      <div></div>
+      <div style="float:left;margin-top:50px;height:auto;width:99%;background-color:#00ffff">
+         <h2 style="text-decoration:underline">ABOUT ME </h2>
+         <div>
+            <p>
+               I am anjali from <em><b>solapur</b></em><br>
+               I have completed my engineering from <strong>N.K.Orchid college of engg and tech</strong><br>
+            </p>
+         </div>
+         <h3 style="text-decoration:underline">FIND ME ON THE WEB</h3>
```

Comment:
can you use alternative html tag for underline text

Reply:
no sir i m adding css style:text-dcoration:underline on heading alternative tag is <u>my underline text</u>